code/modules/antagonists/heretic/knowledge/moon_lore.dm

```diff
@@ -124,7 +126,6 @@
 	gain_text = "At the head of the parade he stood, the moon condensed into one mass, a reflection of the soul."
 	next_knowledge = list(
 		/datum/heretic_knowledge/blade_upgrade/moon,
```

Comment:
Removals should be done with commenting, do these for other files too
```suggestion
		/datum/heretic_knowledge/blade_upgrade/moon,
		// /datum/heretic_knowledge/reroll_targets, // BUBBER EDIT REMOVAL
```

Reply:
Did it for all in newest commit!